pkg/result/ignore.go

```diff
@@ -178,7 +179,43 @@ func (c *IgnoreConfig) MatchSecret(secretID, filePath string) *IgnoreFinding {
 }
 
 func (c *IgnoreConfig) MatchLicense(licenseID, filePath string) *IgnoreFinding {
-	return c.Licenses.Match(licenseID, filePath, nil)
+	if f := c.Licenses.Match(licenseID, filePath, nil); f != nil {
+		return f
+	}
+
+	var licenseNotMatch bool
+	matchLicenses := func(expr expression.Expression) expression.Expression {
+		// If one of parts of the expression doesn't match - skip check for the rest of the expression
+		if licenseNotMatch {
+			return expr
+		}
+
+		if e, ok := expr.(expression.CompoundExpr); ok && e.Conjunction() != expression.TokenWith {
+			// Check only license with `WITH` operator as single license
+			if e.Conjunction() != expression.TokenWith {
+				return e
+			}
+		}
+
+		if !expr.IsSPDXExpression() || c.Licenses.Match(expr.String(), filePath, nil) == nil {
+			licenseNotMatch = true
+		}
+		return expr
+	}
+
+	_, err := expression.Normalize(licenseID, matchLicenses)
```

Comment:
I think we need to normalize license before find ignore rule.
```suggestion
	_, err := expression.Normalize(licenseID, licensing.NormalizeLicense, matchLicenses)
```